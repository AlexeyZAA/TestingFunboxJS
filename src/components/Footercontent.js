import React from 'react'

const Footercontent = ({ firstName, lastName }) => (
  <div>
    Автор:{' '}
    <h3>
      {' '}
      {firstName} {lastName}{' '}
    </h3>
    .
  </div>
)

export default Footercontent