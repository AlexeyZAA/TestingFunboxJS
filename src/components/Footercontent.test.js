import React from 'react'
import { render } from '@testing-library/react'
import Footercontent from './Footercontent'
/* eslint-env jest */

test('тест снимок небольшого вывода футера', () => {
  const { asFragment  } = render(<Footercontent firstName={'Имя'} lastName={'Фамилия'}/>)

  expect(asFragment()).toMatchSnapshot()
})
