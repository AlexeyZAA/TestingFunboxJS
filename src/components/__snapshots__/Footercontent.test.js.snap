// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`тест снимок небольшого вывода футера 1`] = `
<DocumentFragment>
  <div>
    <h3>
      Автор:Имя Фамилия
    </h3>
  </div>
</DocumentFragment>
`;
